docs/wallet-integration-guide/src/exchange-integration/extensions.rst

```diff
@@ -6,16 +6,94 @@ Integration Extensions
+   against a single node being compromised. Note that it does not
+   provide availability in case of compromise though: If one node is
+   compromised or down, transactions will be rejected as you will not
+   be able to reach 2 valid confirmations. To be able to run
+   transactions again you either need to bring both nodes back online
+   or change the hosting nodes.
+2. 3 confirming nodes with a threshold of 2. This extends the previous
+   setup to also provide availability in case one of the nodes goes
+   down or gets compromised as you can still reach 2 valid
+   confirmations.
+
+Party Setup
+^^^^^^^^^^^
+
+.. TODO:: https://github.com/hyperledger-labs/splice-wallet-kernel/issues/272 Update this when wallet SDK support is available
+
+As part of the :ref:`initial party setup <create-an-external-party>`_,
+you can specify multiple multiple confirming participants and a
+threshold as needed. Note that at this point, the wallet SDK library
+does not yet support this so you must go directly through the Canton
+APIs. This is expected to change soon. Until then, the easiest way to do so at the moment is through the Canton
+console. You can find a full reference for all required steps in the
+`integration test
+<https://github.com/digital-asset/canton/blob/3c9ac9891c03cb06303736d7224bcc01dbd50084/community/app/src/test/scala/com/digitalasset/canton/integration/tests/jsonapi/ExternalPartyLedgerApiOnboardingTest.scala#L183>`_. Note
+in particular that you must sign the ``PartyToParticipant`` mapping
+not just by your party's key but also by all confirming
+participants. This is accomplished through the
+``participant2.topology.transactions.authorize`` step in the test.
+
+DAR Management
+^^^^^^^^^^^^^^
+
+Any DAR that you upload, both as part of the initial setup but also
+whenever you upload newer versions to upgrade an existing package,
+must be uploaded to all confirming nodes of the party.
+
+Reading Data and Submitting Transactions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use any of the confirming nodes for reading data of a party
+hosted on them. However, offsets are not comparable across nodes so it
+is recommended that to run Tx History Ingestion against the same node
+under normal operations. If you do need to switch nodes, you can do so
+following the same procedure used for `restoring a validator from a
+backup <validator_backup_restore>`_ to resynchronize Tx History
+Ingestion against the offsets of the new node.
+
+Preparation and execution of transactions can also be done against any
+of the confirming nodes of the party. However, `Command Deduplication
+<https://docs.digitalasset.com/build/3.3/sdlc-howtos/applications/develop/command-deduplication.html>`_
+is only performed by the executing node so if you submit across nodes you cannot rely on it.
+
+.. TODO:: Link to recommended deduplication strategy https://github.com/hyperledger-labs/splice-wallet-kernel/issues/423
```

Comment:
I currently do not recommend command dedup at all. Instead UTXO-based dedup is used jointly with the record-time timeout.

Reply:
My worry is someone searches for deduplication in the Canton docs finds that and then gets surprised it doesn't work. So calling it out as a limitation here seemed prudent. Let me make it clearer that we recommend against using it.